src/Bundle/Resources/skeleton/Factory.tpl.php

```diff
@@ -10,9 +10,11 @@
 use Zenstruck\Foundry\Proxy;
 
 /**
+ * @extends ModelFactory<<?= $entity->getShortName() ?>>
+ *
  * @method static <?= $entity->getShortName() ?>|Proxy createOne(array $attributes = [])
```

Comment:
should it be `&Proxy` rather than `|Proxy`? After generating factories, PHPStan complains with errors unless I change it to intersection type instead of union.

Reply:
This is a tough one. PHPStorm 2021.3.2 doesn't autocomplete the object when an intersection is used.

I think the proper type should be `Proxy<<?= $entity->getShortName() ?>>` but that doesn't autocomplete either...